api/server/runner_fninvoke.go

```diff
@@ -44,6 +45,12 @@ func (s *syncResponseWriter) Status() int          { return s.status }
 
 // handleFnInvokeCall executes the function, for router handlers
 func (s *Server) handleFnInvokeCall(c *gin.Context) {
+	if c.Request.Method != http.MethodPost {
+		handleErrorResponse(c, models.NewAPIError(http.StatusMethodNotAllowed,
+			errors.New("Method not allowed, only HTTP POST supported.")),
+		)
```

Comment:
wanna make `models.ErrInvokePostOnly` in `api/models/error.go` ? could test against the error string too, which would be nice

Reply:
Done.